airflow/bin/cli.py

```diff
@@ -391,8 +339,22 @@ def run(args, dag=None):
     ti = TaskInstance(task, args.execution_date)
     ti.refresh_from_db()
 
+    logger = logging.root
+    logger.handlers = []
+    if args.raw:
+        # Output to STDOUT for the parent process to read and log
+        logging.basicConfig(
```

Comment:
The comment above indicates it needs to be output to STDOUT for parent to read the log. Does it make sense to pass this logic to other places?